src/utils/address.cljs

```diff
@@ -101,30 +101,39 @@
 
 (defn eip-3770-address?
   "Checks if address follows EIP-3770 format which is default for Status"
-  [s]
-  (re-find regx-eip-3770-address s))
+  [address]
+  (re-find regx-eip-3770-address address))
 
 (defn supported-address?
   [s]
   (boolean (or (eip-3770-address? s)
                (metamask-address? s))))
 
-(defn metamask-address->status-address
+(defn metamask-address->eip-3770-address
   [metamask-address]
   (when-let [[_ address metamask-network-suffix] (split-metamask-address metamask-address)]
     (if-let [status-network-prefix (eip-155-suffix->eip-3770-prefix metamask-network-suffix)]
       (str status-network-prefix address)
       address)))
 
+(defn extract-address-without-chains-info
+  [address]
+  (re-find regx-address-contains address))
+
+(defn metamask-address->status-address
+  [eip-3770-address]
+  (extract-address-without-chains-info eip-3770-address))
+
+(defn eip-3770-address->status-address
```

Comment:
What's the difference between eip-3770-address and status-address?
If it is the address without the prefixes maybe better to call it `plain-address`?

Reply:
Currently status address is plain address. I left the naming `status-address` to leave it intact in case in the future we will decide that status address is something else. But I have nothing against naming it `plain-address` if you feel this would be more informative. Wdyt?

Reply:
IMO is quite confusing naming it `status-address` while is just a typical Ethereum address. Not sure what the rest of the team think? @shivekkhurana @mohsen-ghafouri @ulisesmac @smohamedjavid  any thoughts on this?

Reply:
- Saw this map in the test too `eip-3770-to-status` 
- Also saw `metamask-address` in the code base. 

I'll suggest to just call it `address`, `wallet-address` or `eth-address`. The term `basic` doesn't add any meaning.